### ISSUE TITLE
What is the state of m_b_union_all_dimensions in OperatorUnionCursor

### ISSUE THREAD

**Issue by tdcmeehan** [Esri/geometry-api-java#199]
I was looking at the source for `OperatorUnionCursor`, and found `m_b_union_all_dimensions` flag.  I am curious what the intent of it is?  It seems to always be set to false.  Is the intent to one day allow support for `OperatorUnionCursor` to support multiple dimensions (as opposed to current behavior, which drops the lowest dimensions)?  If that's what it was intended for, do we think it's feasible to modify this code to allow setting it to true when we want to allow support for unioning multiple dimensions of geometries with this operator?

**Comment by stolstov** [comment]
Yes, this is unfinished code, I forgot to remove it. We don't have a requirement to support this behavior in the OperatorUnion. The only place where this behavior was needed is OGCGeometry.union, where it was done in a custom way.
